TractorMod/Framework/Config/ScytheConfig.cs

```diff
@@ -29,5 +29,8 @@ internal class ScytheConfig
 
         /// <summary>Whether to clear weeds.</summary>
         public bool ClearWeeds { get; set; } = true;
+
+        /// <inheritdoc/>
```

Comment:
Whoops, used this out of habit.  (Not used elsewhere).  I can revert to a copy/paste of the interface if you want.

(I'd recommend you look into using this everywhere to cut down on duplicate comments.)

Reply:
I went ahead and reverted it.  Still, strongly suggest using <inheritDoc>

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#inheritdoc

Reply:
The code does use `inheritdoc` in a lot of places already, so it'd be fine to use it here since you added the interface. I'll do that before merging though.